cmd/launcher/launcher.go

```diff
@@ -243,6 +244,11 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 		runGroup.Add(launcherUpdater.Execute, launcherUpdater.Interrupt)
 	}
 
+	if (opts.KolideServerURL == "k2device-preprod.kolide.com" || opts.Debug) && (runtime.GOOS == "darwin" || runtime.GOOS == "linux") {
```

Comment:
I added this because I'm often running this locally on my machine against a local kolide server. However, I can understand why you would want to remove this. How about checking against "localhost:3443" ?

Reply:
Adding a localhost check seem okay. And maybe heroku, I don't love it, but it might be an okay stopgap.

But we do suggest folks run launcher in debug mode, and I wouldn't want systray to pop up.